test/server/cards/01_SOR/units/BobaFettDisintegrator.spec.js

```diff
@@ -0,0 +1,115 @@
+describe('Boba Fett Disintegrator', function() {
+    integration(function() {
+        describe('Boba Fett Disintegrator\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['vambrace-grappleshot'],
+                        groundArena: ['boba-fett#disintegrator'],
+                    },
+                    player2: {
+                        leader: 'luke-skywalker#faithful-friend',
+                        hand: ['lom-pyke#dealer-in-truths', 'consular-security-force', 'entrenched'],
+                    }
+                });
+            });
+
+            it('while attacking an exhausted unit that didn\'t enter play this round, deal 3 damage to the defender.', function () {
+                // Case 1 attacking a ready card
+                this.player1.passAction();
+                this.player2.clickCard(this.consularSecurityForce);
+                this.player1.passAction();
+
+                this.player2.clickCard(this.lukeSkywalker);
+                this.player2.clickPrompt('Deploy Luke Skywalker');
+                this.moveToNextActionPhase();
+
+                this.player1.clickCard(this.bobaFett);
+                this.player1.clickCard(this.consularSecurityForce);
+
+                // check board state
+                expect(this.consularSecurityForce.damage).toBe(3);
+                expect(this.bobaFett.damage).toBe(3);
+
+                // reset board state
+                this.bobaFett.damage = 0;
```

Comment:
fellarmir had a good idea for making this a little simpler by creating a reset method so we can have less code being repeated: https://github.com/SWU-Karabast/forceteki/blob/965dae3d78844c0cee7dfff102b6f76c623ad6c3/test/server/cards/01_SOR/leaders/BobaFettCollectingTheBounty.spec.js#L22

Reply:
Added reset function to the tests to make it simpler.